.github/workflows/release-pr.yml

```diff
@@ -1,9 +1,10 @@
+name: Open Release PR for review
```

Comment:
I moved `name` to the top to be consistent with other workflow files, with the expectation that the template I'm working on will have name first.